### ISSUE TITLE
"Preview" Flag on VSIX Manifest not Manifesting to Marketplace

### ISSUE THREAD

**Issue by CalvinAllen** [microsoft/vsmarketplace#905]
I have an extension that specifies "Preview" is true in the vsix manifest file, yet the marketplace doesn't seem to be honoring this flag as I upload new versions.

[Manifest File](https://github.com/CodingWithCalvin/VS-CouchbaseExplorer/blob/main/src/CodingWithCalvin.CouchbaseExplorer/source.extension.vsixmanifest
)

[Marketplace Entry](https://marketplace.visualstudio.com/items?itemName=CodingWithCalvin.VS-CouchbaseExplorer
)

<img width="1704" alt="Screenshot 2024-05-15 at 6 50 33 PM" src="https://github.com/microsoft/vsmarketplace/assets/41448698/6b66e56b-8121-49d6-863d-997758958856">

<img width="1704" alt="Screenshot 2024-05-15 at 6 50 46 PM" src="https://github.com/microsoft/vsmarketplace/assets/41448698/97003ade-13a0-4477-8269-34f14fcf572b">

I deploy using the VSIXPublisher utility and [a publishing manifest file](https://github.com/CodingWithCalvin/VS-CouchbaseExplorer/blob/main/.github/workflows/publish.yml)

**Comment by madhurivadaligithub** [comment]
Hi @CalvinAllen 

Thank you for reaching out to the VS Marketplace Team. We are investigating the issue and will get back to you with an update promptly.

Regards,
VS Marketplace Team

**Comment by Neelima10735584** [comment]
Hi,
 could you please go through the below public document please try again and let us know if still facing the issue 
 
 [New Preview label for Visual Studio extensions - Visual Studio Blog (microsoft.com)](https://devblogs.microsoft.com/visualstudio/new-preview-label-for-visual-studio-extensions-2/)
 
 Regards,
VS Marketplace Team

**Comment by CalvinAllen** [comment]
@madhurivadaligithub @Neelima10735584 It does not work. All the document references is adding the Preview tag/flag into the vsix manifest which I have (https://github.com/CodingWithCalvin/VS-CouchbaseExplorer/blob/main/src/CodingWithCalvin.CouchbaseExplorer/source.extension.vsixmanifest#L12)

**Comment by Neelima10735584** [comment]
Hi Calvin,
We have observed that The Preview Functionality is working correctly from manage User Interface (UI).
 please upload through marketplace UI to unblock yourself, we are actively working on vsixpublisher to repro the issue and keep you posted on this progress.
 
Regards,
VS Marketplace Team

**Comment by Neelima10735584** [comment]
Hi Calvin,
 
It appears that your extension is not available in the marketplace. Are you still encountering issues when trying to upload it through UI also?
 
Please let me know if you need any assistance or further information.
 
Regards,
VS MarketPlace Team

**Comment by CalvinAllen** [comment]
No, I have unpublished it for the time being because publishing every single update through the UI is not a solution for me.

**Comment by Neelima10735584** [comment]
Thank you for your update.We understand your concern .We are investigating the issue and will get back to you with a resolution soon.

Regards,
VS MarketPlace Team

**Comment by Neelima10735584** [comment]
Hi Calvin,
 
We are unable repro the issue, and also we have not received any requests on same issue, we need few details from you 
could you please share the step-by-step process of you are following to upload this preveiw extension to Marketplace.
 
Additionally, if you need any assistance related to vsix publisher, please reach out to them through the github page: https://github.com/microsoft/VSExtensibility/issues.

Regards,
VS MarketPlace Team

**Comment by Neelima10735584** [comment]
Hi Calvin,

We are closing the issue since there has been no updates from you. If the issue still persists, please feel free to reopen or create a new issue.

Regards,
VS MarketPlace Team
